pages/database-management/monitoring.mdx

```diff
@@ -7,50 +7,102 @@ import { Callout } from 'nextra/components'
-databases as well. Memgraph allows monitoring the database via WebSocket server and a
-HTTP server. 
+Monitoring of applications and databases is essential to be able to track the
+load, resources, logs, transactions, latency etc. 
 
-## Monitoring via WebSocket server
+Memgraph currently supports:
+
+- Logs tracking via WebSocket server 
+- Monitoring via HTTP server (Enterprise Edition)
+
+Log tracking is helpful for debugging purposes and monitoring the database
+operations.  
+ 
+In the Enterprise Edition, on top of log tracking,  Memgraph allows tracking
+information about transactions, query latencies, snapshot recovery latencies,
+triggers, bolt messages, indexes, etc., using an HTTP server.
+
+
+## Logs tracking via WebSocket server 
+
+Connect to Memgraph's logging server via WebSocket to forward logs to all the
+connected clients.
+
+
+### Log messages
+
+Each log that is written to the [log file](logs.mdx#logs) is forwarded to the
+connected clients in the following format:
+
+```json
+{
+  event: "log",
+  level: "trace"|"debug"|"info"|"warning"|"error"|"critical",
+  message: "<log-message>"
+}
+```
 
-Connect to Memgraph's monitoring server via WebSocket to forwarded logs to all
-the connected clients.
 
 ### Connection
 
-To connect to Memgraph's WebSocket server use the following URL:
+To connect to Memgraph's WebSocket server, use the following URL:
 
 ```plaintext
 ws://host:port
 ```
 
-The default host is `localhost` but it can be changed using the
-`--monitoring-host` configuration flag.  
-The default port is `7444` but it can be changed using the `--monitoring-port`
+The default host is `0.0.0.0`, but it can be changed using the
+`--monitoring-address=` configuration flag.  
+
+The default port is `7444`, but it can be changed using the `--monitoring-port`
 configuration flag.
 
+To connect to Memgraph's WebSocket server using the default configuration and
+Python client, you can use the following code snippet:
+
+```Python
+import asyncio
+import websockets
+
+HOST = 'localhost'
+PORT = '7444'
+
+# The WebSocket URL
+ws_url = f'ws://{HOST}:{PORT}'
+
+async def read_logs():
+    async with websockets.connect(ws_url) as websocket:
+        print(f"Connected to Memgraph at {ws_url}")
+        try:
+            # Keep reading messages (logs) from the server
+            while True:
+ log_message = await websocket.recv()
+                print(log_message)
+        except websockets.exceptions.ConnectionClosed:
+            print("Connection to Memgraph closed.")
+
+asyncio.run(read_logs())
+```
+
+The code structure is similar in other languages, depending on the WebSocket
+library used.
+
 If you want to connect with a WebSocket Secure (WSS) connection. Set the
-following configuration flags to enable
-an SSL connection: `--bolt-cert-file` and `--bolt-key-file`.
+following configuration flags to enable an SSL connection: `--bolt-cert-file`
+and `--bolt-key-file`.
```

Comment:
smth is not right here, fix the sentence

Reply:
Done.